plaso/parsers/dsv_parser.py

```diff
@@ -50,9 +50,6 @@ class DSVParser(interface.FileObjectParser):
   # file to see if it confirms to standards.
   _MAGIC_TEST_STRING = b'RegnThvotturMeistarans'
 
-  # Maximum supported file size of 16 MiB.
```

Comment:
This is a refactoring artifact, and should have been removed in https://github.com/log2timeline/plaso/pull/2158